week6/Products.cs

```diff
@@ -0,0 +1,48 @@
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace HackYourFuture.Week6;
+
+
+public interface IProductRepository
+{
+    Task<Product> GetProduct(int id);
+    Task<Product> PostProduct(Product product);
```

Comment:
You could simplify the name here by using `Get` instead of `GetProduct` and `Create` instead of `PostProduct`